### PR TITLE
UI wallet adaptor page

### LINKED ISSUE #99: Implement the Wallet Adapter page
-Create the wallet adapter information page
-Add your changes to the dev-frontend branch

### PR DESCRIPTION
**Pull request by saketsingh0078**
**Issue Number:** Fixes #99
Implement the basic UI of wallet Adaptor Page 

## 🛠️ Type of Change

- [ ] 🐛 Bug fix
- [x] ✨ New feature
- [ ] 🚨 Breaking change
- [ ] 📚 Documentation update

## ✅ Checklist

To ensure a smooth review process, please check off each item as you complete it:

- [x] **Code Style:** My code adheres to the project’s style guidelines.
- [x] **Self-Review:** I have reviewed my own code and made improvements.

https://github.com/user-attachments/assets/9bd961b8-f8c4-49dd-8adf-03da10e7172c

### CODE REVIEW

**Comment by cb7chaitanya** [comment]
Can you make the topbar mobile responsive, rest is fine, that seems a bit ugly when in mobile view @saketsingh0078

**Comment by saketsingh0078** [comment]
@cb7chaitanya ok sure

**Comment by saketsingh0078** [comment]
@cb7chaitanya I use the existing topbar from the wallet page. Look like the topbar of wallet Page is changed then it accordingly to wallet Page topbar.

**Review by ghost — COMMENTED** [review]
Do we need a search bar ??

File: src/app/wallet-adapter/page.tsx [hunk 1]
```diff
@@ -1,10 +1,157 @@
...
+                icon: DollarSign,
+                title: 'Low Fees',
+                description:
+                  'Minimize transaction costs for your users with optimized fee structures.',
+              },
+              {
+                icon: Star,
+                title: 'Customizable UI',
+                description:
+                  'Fully customizable UI to match your brand and user experience.',
```
Comment (cb7chaitanya) [inline src/app/wallet-adapter/page.tsx]: We can really keep the code more presentable by keeping this array of objects above the return statement, makes code more readable

File: src/app/wallet-adapter/page.tsx [hunk 2]
```diff
@@ -1,10 +1,157 @@
...
+      <section className="py-20 px-4 bg-secondary">
+        <div className="max-w-4xl mx-auto text-center">
+          <h2 className="text-3xl font-bold mb-12">What Our Users Say</h2>
+          <div className="grid grid-cols-1 md:grid-cols-2 gap-8">
+            {[
+              {
+                name: 'Alex Johnson',
+                role: 'Solana Developer',
+                quote:
+                  "TipLink Wallet Adapter has significantly reduced our user onboarding time. It's a game-changer!",
```
Comment (cb7chaitanya) [inline src/app/wallet-adapter/page.tsx]: Same for this, why add fake testimonials, we can add something else other than this on this page, right?

Comment (saketsingh0078) [reply src/app/wallet-adapter/page.tsx]: right. I will add something.

File: src/app/wallet-adapter/page.tsx (hunk 1 above)
Comment (saketsingh0078) [reply src/app/wallet-adapter/page.tsx]: ok.

**Comment by saketsingh0078** [comment]
@cb7chaitanya make the changes u suggested.

**Comment by cb7chaitanya** [comment]
Send the final screenshot of the page @saketsingh0078

**Comment by saketsingh0078** [comment]
@cb7chaitanya

https://github.com/user-attachments/assets/28f9ed31-72d7-4b70-91bf-7c73f565a356

**Comment by cb7chaitanya** [comment]
Cool, Merging it @saketsingh0078

**Pull request merged**